tests/builtin.yml

```
@@ -1 +1,20 @@
 ---
+parameters:
+  keycloak:
+    extraEnv:
+      "FOO":
+        value: "bar"
+    extraInitContainers:
+      "theme-provider":
```

Those quotes aren't required AFAIK
```suggestion
      FOO:
        value: bar
    extraInitContainers:
      theme-provider:
```

---

Yes probably not. I'm generally use quotes a bit too liberally :)

But in general I think this parameter structure is a lot better then more strings, and the postprocessing step is not too bad.